vanilla-tests/src/main/java/fixtures/streamstyleserialization/Arrays.java

```diff
@@ -0,0 +1,318 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package fixtures.streamstyleserialization;
```

Comment:
We should update the tests to run against the `fixtures.streamstyleserialization` types as well.